google-cloud-storage/src/main/java/com/google/cloud/storage/transfermanager/DownloadResult.java

```diff
@@ -27,7 +28,9 @@
 import org.checkerframework.checker.nullness.qual.MonotonicNonNull;
 import org.checkerframework.checker.nullness.qual.NonNull;
 
+@BetaApi
 public final class DownloadResult {
+  @BetaApi
```

Comment:
Not strictly necessary as this field is package private, but it doesn't hurt anything either.